src/main/java/org/javacord/bot/commands/InviteCommand.java

```diff
@@ -0,0 +1,20 @@
+package org.javacord.bot.commands;
+
+import de.btobastian.sdcf4j.Command;
+import de.btobastian.sdcf4j.CommandExecutor;
+import org.javacord.api.entity.channel.TextChannel;
+import org.javacord.api.entity.message.embed.EmbedBuilder;
+import org.javacord.bot.Constants;
+
+public class InviteCommand implements CommandExecutor {
+
+    @Command(aliases = {"!invite"}, async = true)
+    public void onCommand(TextChannel channel) {
+        EmbedBuilder embed = new EmbedBuilder()
+                .setThumbnail(getClass().getClassLoader().getResourceAsStream("javacord3_icon.png"), "png")
+                .setColor(Constants.JAVACORD_ORANGE)
+                .addField("Javacord Server link:", "https://discordapp.com/invite/0qJ2jjyneLEgG7y3");
```

Comment:
To be consistent with the design of the other commands, there shouldn't be a `:`.
I would also prefer `Invite Link` instead of `Javacord Server link`.